src/deadlineresolver.js

```diff
@@ -208,7 +208,7 @@ class DeadlineResolver {
       debug('Resolved taskId: %s, by deadline', taskId);
 
       // Update dependency tracker
-      await this.dependencyTracker.resolveTask(taskId, 'exception');
+      await this.dependencyTracker.resolveTask(taskId, task.taskGroupId, 'exception');
```

Comment:
@imbstack I think this is a case where we have to add it to the message again...

The deadline message serves 3 purposes:
 A) Resolve old tasks
 B) Resolve tasks that failed to create properly
 C) Clean up TaskGroupActiveSet and publish message about task-group finished...